css/layout-css/news-feed-style.upload.css

```diff
@@ -558,6 +558,8 @@
   opacity: 1;
   margin-bottom: 40px;
   transition: box-shadow 0.15s ease-out;
+  -webkit-transform: translate3d(0, 0, 0);
```

Comment:
@tonytlwu Did you test this change on:
1- One or two layouts from production? (copying the HTML from a layout on production)
2- Inside an accordion?
3- Expanding and collapsing the card details?

Reply:
I haven’t tested the first one. Let me give that a go 👍🏼